map_it_test.go

```diff
@@ -1047,7 +1047,7 @@ func TestMap_SetTTL(t *testing.T) {
 		ctx := context.Background()
 		targetValue := "value"
 		it.Must(m.Set(ctx, "key", targetValue))
-		if err := m.SetTTL(ctx, "key", 20*time.Second); err != nil {
+		if err := m.SetTTL(ctx, "key", 2*time.Second); err != nil {
```

Comment:
Why the change?

Reply:
I think this takes too much time on test pipeline. Am I mistaken?

Reply:
2 seconds is not enough for the TTL behaviour. That will end up with a flaky test.

Reply:
Got it, I was not aware of that. Asking just for curiosity, is this a documented behavior?
I would love to check if there is a document for these in the future, before making such changes.